### DISCUSSION TITLE
Cloud save not working

### DISCUSSION THREAD

**Discussion by Deejay923** [moraroy/NonSteamLaunchers-On-Steam-Deck#153] [Q&A] [answered]
If I open a game within the Epic Games app everything works well no issues. But if I add a game from epic to my steam library with Boilr I do not have access to my cloud saves. I enabled the option to force the launch through epic games in Boilr, did not resolve the issue. Anyone know of a workaround? Would really like to able to launch my games through my steam library so I can set per game controls for each game.

**Comment by moraroy** [comment, marked as answer]
@Deejay923 were you able to get this fixed? Some games don't have cloud save functionality just FYI. Using Boilr to get your shortcuts should work though because you games will open up with epic in the background as you have said.

**Reply by Deejay923** [reply]
Yea, idk what was going on initially but I set everything up again and it was good to go. Been working fine ever since. Thank you!
